Source/AwsCommonRuntimeKit/mqtt/Mqtt5Packets.swift

```diff
@@ -460,29 +610,30 @@ public class DisconnectPacket: CStruct {
     }
 
     static func convertFromNative(_ from: UnsafePointer<aws_mqtt5_packet_disconnect_view>?) -> DisconnectPacket? {
-        if let _from = from {
-            let disconnectView = _from.pointee
-            guard let reasonCode = DisconnectReasonCode(rawValue: Int(disconnectView.reason_code.rawValue))
-            else { fatalError("aws_mqtt5_packet_disconnect_view from native missing a reason code.") }
-
-            let sessionExpiryInterval = convertOptionalUInt32(disconnectView.session_expiry_interval_seconds)
-            let sessionExpiryIntervalSeconds: TimeInterval? = sessionExpiryInterval.map { TimeInterval($0) }
-            let reasonString = convertAwsByteCursorToOptionalString(disconnectView.reason_string)
-            let serverReference = convertAwsByteCursorToOptionalString(disconnectView.reason_string)
-            let userProperties = convertOptionalUserProperties(
-                count: disconnectView.user_property_count,
-                userPropertiesPointer: disconnectView.user_properties)
-
-            let disconnectPacket = DisconnectPacket(
-                reasonCode: reasonCode,
-                sessionExpiryInterval: sessionExpiryIntervalSeconds,
-                reasonString: reasonString,
-                serverReference: serverReference,
-                userProperties: userProperties
-            )
-            return disconnectPacket
+        guard let from = from else {
+            return nil
         }
-        return nil
+        let disconnectView = from.pointee
+        guard let reasonCode = DisconnectReasonCode(rawValue: Int(disconnectView.reason_code.rawValue))
+        else { fatalError("aws_mqtt5_packet_disconnect_view from native missing a reason code.") }
+
+        let sessionExpiryInterval = convertOptionalUInt32(disconnectView.session_expiry_interval_seconds)
+        let sessionExpiryIntervalSeconds: TimeInterval? = sessionExpiryInterval.map { TimeInterval($0) }
+        let reasonString = convertAwsByteCursorToOptionalString(disconnectView.reason_string)
```

Comment:
Don't need this function, Utilities.swift already has a helper extension. You can just do `disconnectView.reason_string.toOptionalString()`.

Reply:
This change will be addressed in a separate PR.